test-data/unit/fixtures/object_with_init_subclass.pyi

```diff
@@ -59,3 +59,4 @@ class dict(Mapping[KT, VT]):
     @overload
     def get(self, k: KT, default: Union[KT, T]) -> Union[VT, T]: pass
     def __len__(self) -> int: ...
+import _typeshed
```

Comment:
Let's move this to the very top (even though that may affect line numbers in 1-2 existing tests, that's simple enough)